src/Components/Template.tsx

```diff
@@ -0,0 +1,196 @@
+
+import oceanMusic from "../Audio/ocean.mp3";
+import neutral from "../Audio/summer.mp3";
+import extreme from "../Audio/extreme.mp3";
+import jazzy from "../Audio/jazzy.mp3";
+import gentle from "../Audio/gentle.mp3";
+import uplifting from "../Audio/uplifting.mp3";
+
+// duration is the length of each media displayed in video
+// if there are too many photos, then mb dont show all? (for now)
+// length is the length of the final video
+// durations and lengths in seconds
```

Comment:
I'm a bit confused about the difference between `length` and `duration`
I think it would be clearer if `duration` was always used when referring to time, and we replace `length: "short"` (etc) with `name` or `type`. WDYT?